jenkins/shared-library/vars/monorepoControllerPipeline.groovy

```diff
@@ -0,0 +1,171 @@
+import java.nio.file.Paths
```

Comment:
i see this is controlled by controller.groovy, but how/when is that triggered? 

each branch/pr creation?

Reply:
yea, the controller will be triggered for every push/pr.  Then it will figure out which CI job to run.